src/util/index.ts

```diff
@@ -89,15 +89,15 @@ export function first(
  *
  * @param {Element} [elem] The element to query (defaults to the `window.document`)
  * @param {string} selector The selector to match
- * @returns {Element | null} An `Element` or `null` if no match
+ * @returns {Element | undefined} An `Element` or `undefined` if no match
```

Comment:
`null` is the DOM way of indicating an element or attribute is not found. Curious why it's `undefined` here.

Reply:
I was trying to have a consistent API across the functions in which only `undefined` was used, the "billion dollar null mistake" and all that. But take this point and have made changes so that both `attr` and `first`, return `null` and that `create` ignores both `null` and `undefined`.